photon-ml/src/integTest/scala/com/linkedin/photon/ml/data/DataValidatorsIntegTest.scala

```diff
@@ -25,7 +25,7 @@ import org.testng.Assert
 import org.testng.annotations.{DataProvider, Test}
 
 /**
- * @author nkatariy
+ *
  */
```

Comment:
The TODO would be about adding documentation to the class. As for the empty comment block, we can keep it or remove it.

This applies to all of the locations @joshvfleming mentioned - adding a TODO comment about documentation.